packages/common/src/types/space.ts

```diff
@@ -49,7 +49,7 @@ export type CreateSpace = {
 
 export type UpdateSpace = {
     name: string;
-    isPrivate: boolean;
+    isPrivate?: boolean;
```

Comment:
what's the default if undefined? IIRC `isPrivate: unefined` means `isPrivate: true`

Reply:
correct, it's `true` - checked update method in SpaceModel